Videos/Smart Configurator Videos_Common.htm

```diff
@@ -7,54 +7,80 @@
         <meta name="description" content="List of videos on how to use the Renesas Smart Configurator tool">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" type="text/css" href="./styles.css">
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+        <link href='http://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
     </head>
 
     <body>
         <div class="wrapper">
-            <div class="sidebar">
+            <div class="header-content">
                 <div class="logo-container">
                     <img class="logo" src="./header-w.svg" alt="Renesas Electronics Corporation">
                 </div>
-                <h2>Content</h2>
-                <ul>
-                    <!-- Add videos list here -->
-                    <li><a href="#_Toc87439352">Smart Configurator Introduction</a></li>
-                    <li><a href="#_Toc87439353">How to Create a Smart Configurator project in e² studio</a></li>
-                </ul>
-            </div>
-            <div class="main-content">
                 <div class="header">
-                    <h1>Smart Configurator Videos"</h1>
-                </div>
-                <div class="video-wrapper">
-                    <!-- repeat video-contents class -->
-                    <div class="video-contents">
-                        <a id="_Toc87439352"></a>
-                        <iframe src="https://players.brightcove.net/5260471205001/RyxO3haSE_default/index.html?videoId=5819365850001" 
-                        allowfullscreen=allowfullscreen></iframe>
-                        <h3>Smart Configurator Introduction</h3>
-                        <div class="video-description">
-                            <p></p>
-                            <p>Date of Upload: 2018-08-08</p>
-                        </div>
-                        <a href="https://www.renesas.com/video/smart-configurator-introduction">View on Renesas website</a>
-                    </div>
-                    <div class="video-contents">
-                        <a id="_Toc87439353"></a>
-                        <iframe src="https://players.brightcove.net/5260471205001/RyxO3haSE_default/index.html?videoId=6286881148001" 
-                        allowfullscreen=allowfullscreen></iframe>
-                        <h3>How to Create a Smart Configurator project in e² studio</h3>
-                        <div class="video-description">
-                            <p>This video demonstrates how to create a Smart Configurator project in e² studio.</p>
-                            <p>Date of upload: 2021-12-15</p>
-                        </div>
-                        <a href="https://www.renesas.com/video/how-create-smart-configurator-project-e-studio">View on Renesas website</a>
+                    <h1>Smart Configurator Videos</h1>
+                    <div class="info notification">
```

Comment:
Q: Do we want to show it if it is empty?

Reply:
Right now, this portion is written in HTML. It is possible to use javascript instead to only generate this portion if there are "new" videos. This way, this "New Videos" will not show up for Common Videos